pages/content/reports/2024/index.md

```diff
@@ -0,0 +1,232 @@
+---
+author: "Florian Scholz"
+title: "Open Web Docs Impact and Transparency Report 2024"
+date: "2025-01-30"
+ShowToc: true
+ShowBreadCrumbs: false
+---
+
+## Executive summary
+
+Open Web Docs is taking a look at how our efforts have helped to ensure the long-term health of web platform documentation on critical resources like MDN Web Docs, independently of any single vendor or organization. Founded in January 2021, the year 2024 marked Open Web Docs’ fourth year of operation!
+
+Open Web Docs (OWD) is an [Open Source Collective](https://opencollective.com/open-web-docs) that employs technical writers to publicly document open web technologies. OWD currently employs two full time writers, one compat data engineering contractor, and OWD's director to run the program. Everyone at OWD has extensive experience with Web standards documentation, MDN Web Docs, and browser compatibility data. The OWD team writes new documentation, updates existing documentation, and improves documentation infrastructure together with other organizations such as Mozilla, Google, Microsoft, W3C, Igalia, and other external contributors and volunteers.
+
+In 2024, Open Web Docs:
+
+- Was again the organization that contributed the greatest number of pull requests (PRs) merged to the [mdn/content](https://github.com/mdn/content) (14.1%) and [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data) (52.4%) repositories.
+- Provided 27% of all pull request reviews to the mdn/content repository and 52% of reviews to mdn/browser-compat-data.
+- Started to become a maintainer of the [web-platform-dx/web-features](https://github.com/web-platform-dx/web-features) repo and contributed 14.8% of merged pull requests.
+- Completed major technical writing and documentation engineering projects which were determined and guided by OWD’s [Steering Committee](https://github.com/openwebdocs/project/blob/main/steering-committee/membership-expectations.md). This included among other things:
+  - [Collecting browser compatibility data whenever browsers release a new beta version](#collecting-browser-compatibility-data-whenever-browsers-release-a-new-beta-version)
+  - [Documentation for interoperable and widely available features](#documentation-for-interoperable-and-widely-available-features)
+  - [W3C WebDX web-features mapping of BCD keys into baseline features](#w3c-webdx-web-features-mapping-of-bcd-keys-into-baseline-features)
+  - [Replacing all "true" and "null" versions with real version numbers in BCD](#replacing-all-true-and-null-versions-with-real-version-numbers-in-bcd)
```

Comment:
this one stands out as non very relatable to if you're not deep into BCD. I think formulating it as providing more accurate historical support data (maybe tying it to baseline calculations?) would be more understanable